src/com/vauff/maunzdiscord/commands/IsItDown.java

```diff
@@ -6,8 +6,9 @@
 import sx.blah.discord.handle.impl.events.guild.channel.message.MessageReceivedEvent;
 
 import java.io.IOException;
-import java.net.InetSocketAddress;
-import java.net.Socket;
+import java.net.*;
```

Comment:
Wildcard imports are bad practice if you're not using everything within the package.

Reply:
Don't blame me, blame your own coding style setup :stuck_out_tongue: